api/dispatcher/json.go

```diff
@@ -69,6 +72,12 @@ func (c JSON) Dispatch(reqBytes []byte) []byte {
 
 	c.logger.DebugWithDetails(jsonLogTag, "Deserialized request", req)
 
+	if strings.Contains(strings.ToUpper(fmt.Sprintf("%s", req)), strings.ToUpper("localDiskFlag")) {
```

Comment:
Move this into it's own private method and add comment (in code) to why we are doing that. Along with a comment note saying that in time we will remove.

Also, add a Unit Test :)